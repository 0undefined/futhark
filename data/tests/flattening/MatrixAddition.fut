fun [int] addRows ([int] xs, [int] ys) =
  map(op+, zip (xs,ys))

fun [[int]] addMatricies ([[int]] A, [[int]] B) =
  map (addRows, zip (A,B))

fun [[int]] main([[int]] A, [[int]] B) =
  addMatricies(A,B)
