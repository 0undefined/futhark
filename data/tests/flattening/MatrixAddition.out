[[4,4,4],[7,9,11]]
